### PR TITLE
[NUI] Make Size and NaturalSize as public API

### PR DESCRIPTION
**Pull request by dongsug-song**
### Description of Change ###
[NUI] Make Size and NaturalSize as public API

for your reference =>
http://suprem.sec.samsung.net/confluence/display/GFX/Tizen+5.0+NUI+Inhouse+API+list

### Bugs Fixed ###
N/A

### API Changes ###
http://suprem.sec.samsung.net/jira/browse/TCSACR-179

### Behavioral Changes ###
N/A

### CODE REVIEW

File: src/Tizen.NUI/src/internal/Size.cs [hunk 1]
```diff
@@ -21,13 +21,11 @@
 
 namespace Tizen.NUI
 {
-    // Please do not use! This will be changed to internal class.
     /// <summary>
     /// A three-dimensional size.
     /// </summary>
-    /// <since_tizen> 3 </since_tizen>
+    /// <since_tizen> 5 </since_tizen>
```
Comment (p-czaja) [inline src/Tizen.NUI/src/internal/Size.cs]: This class is provided from TizenFX version 5 so all <since_tizen> tags in this class should be updated.

**Comment by WonyoungChoi** [comment]
close and reopen to invoke checkers again.

**Comment by WonyoungChoi** [comment]
run ABI Checker

**Comment by dongsug-song** [comment]
Dear Mr. p-czaja 
Hello, 
I updated the comment and requested review again at the last week.
(http://suprem.sec.samsung.net/jira/browse/TCSACR-179)
but no response I got, so I merge this now. 
thank you.
best regards,
ssong.

**Pull request merged**